src/redux/categories/catSlice.jsx

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  status: 'under-development',
+};
```

Comment:
- [ ] Please make sure to use an empty array in the initial state as requested in the project requirements.